.jshintrc

```diff
@@ -23,7 +23,7 @@
   "undef"         : true,     // true: Require all non-global variables to be declared (prevents global leaks)
   "unused"        : false,    // true: Require all defined variables be used
   "strict"        : false,    // true: Requires all functions run in ES5 Strict Mode
-  "maxparams"     : 10,       // {int} Max number of formal params allowed per function
+  "maxparams"     : 12,       // {int} Max number of formal params allowed per function
```

Comment:
@ZitaNemeckova totally could remove `CollectionsApi`!  🎨 🎊 we also don't use `Notification` any longer, (now its `EventNotification` and if you need more i could put some work in to get rid of `$state` and `$rootScope`